ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -370,33 +384,47 @@ const GENE_DETAIL_SECTIONS = [
     color: 'pink',
     description: 'RNA-Seq Outlier',
     label: 'RNA-Seq',
-    showDetails: (gene, rnaSeqData) => rnaSeqData && rnaSeqData[gene.geneId],
-    detailsDisplay: (gene, rnaSeqData) => (
-      <div>
-        This gene is flagged as an outlier for RNA-Seq in the following samples
-        <Table basic="very" compact="very">
-          <Table.Header>
-            <Table.Row>
-              <Table.HeaderCell />
-              {RNA_SEQ_DETAIL_FIELDS.map(
-                field => <Table.HeaderCell key={field}>{camelcaseToTitlecase(field).replace(' ', '-')}</Table.HeaderCell>,
-              )}
-            </Table.Row>
-          </Table.Header>
-          <Table.Body>
-            {Object.entries(rnaSeqData[gene.geneId]).map(([individual, data]) => (
-              <Table.Row key={individual}>
-                <Table.HeaderCell>{individual}</Table.HeaderCell>
-                {RNA_SEQ_DETAIL_FIELDS.map(
-                  field => <Table.Cell key={field}>{data[field].toPrecision(3)}</Table.Cell>,
-                )}
-              </Table.Row>
-            ))}
-          </Table.Body>
-        </Table>
-      </div>
+    showDetails: (gene, { rnaSeqData }) => rnaSeqData && rnaSeqData[gene.geneId],
+    detailsDisplay: (gene, { rnaSeqData }) => (
+      <DataTable
+        basic="very"
+        data={rnaSeqData[gene.geneId]}
+        idField="geneId"
+        columns={RNA_SEQ_COLUMNS}
+      />
     ),
   },
+  {
+    color: 'orange',
+    description: 'Phenotype Prioritization',
+    lable: 'PhenotypeGene',
+    showDetails: (gene, { phenotypeGeneScores }) => phenotypeGeneScores && phenotypeGeneScores[gene.geneId],
+    detailsDisplay: (gene, { phenotypeGeneScores }) => (Object.entries(phenotypeGeneScores[gene.geneId]).map(
+      ([tool, data]) => {
+        PHENOTYPE_GENE_SCORE_COLUMNS[tool] = [
```

Comment:
The complexity here is not worth it. Make a single "scores" column, and then have the content for each cell render the scores - something like 
```
format: ({ scores }) => Object.keys(scores).sort().map(scoreName => ([
  <div key={scoreName}>
    <b>{camelcaseToTitlecase(score)}</b>: {scores[scoreName]}
  </div>
])
```

Reply:
Sounds good.